apps/libexec/node.py

```diff
@@ -65,6 +65,7 @@ def cmd_status(args):
 	mentioned = {}
 	pg_oconf_hash = {}
 	pg_conf = {}
+	exit_code = 0
 
 	sinfo = list(get_merlin_nodeinfo(query_socket))
 
```

Comment:
should probably also return non-zero on L74.

Reply:
Attempted to do this below, have a look